src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -169,4 +171,18 @@ public boolean isAppWorkingInBackground() {
         String currentAppPackageName = Runner.getAppPackageName();
         return currentOpenApp.contains(currentAppPackageName);
     }
+
+    @Override
+    public VodqaScreen scrollInDynamicLayer() {
+        driver.waitTillElementIsPresent(byCLanguageTextView);
+        driver.scrollInDynamicLayer(Direction.DOWN);
```

Comment:
The BL should send the direction - either as Direction.UP or Direction.DOWN

Reply:
do i have to pass direction from scenario step OR should i pass it from BL?